### PR TITLE
T1613 - Podman

### PR DESCRIPTION
**Pull request by W00glin**
**Details:**
Tried updating `T1613` to include Podman, since there are more containers utilites outside of just Docker.

**Testing:**
I tried testing this locally on a MacOS VM using the `Invoke-AtomicTest` but kept running int o issues when trying to verifying the prereqs are met.

**Associated Issues:**
None that I can think of, nor see.

I'd also like to add, this is my first ever "real" PR on a project. I'm not a dev, so if I am doing this wrong, apologies. I read through the documentation and watched this [great video about contributing](https://www.youtube.com/watch?v=5MCtee8_s24) and wanted to add some additional checks for other container tools.

If additional info is needed, let me know and I will work on it.

### CODE REVIEW

File: atomics/T1613/T1613.yaml [hunk 1]
```diff
@@ -1,33 +1,66 @@
 ---
 attack_technique: T1613
-display_name: "Container and Resource Discovery"
+display_name: "Docker Container and Resource Discovery"
```
Comment (patel-bhavin) [inline atomics/T1613/T1613.yaml:3]: The display name is correct here. Instead update it lower in the file :

atomic_tests:
- name:  <. >

File: atomics/T1613/T1613.yaml [hunk 2]
```diff
@@ -1,33 +1,66 @@
...
       docker stats --no-stream
       docker inspect $(docker ps -l -q --filter ancestor=t1613)
     cleanup_command: |-
       docker stop t1613_container
       docker rmi -f t1613_container
-    name: sh
\ No newline at end of file
+    name: sh
+---
+attack_technique: T1613
```
Comment (patel-bhavin) [inline atomics/T1613/T1613.yaml:35]: attack_technique, display_name -  are both top level keys. the correct yaml structure would look something like this:

---
attack_technique: T1613
display_name: "Container and Resource Discovery"
atomic_tests:
- name: Docker Container and ResourceDiscovery (update name here not above)
  <............>
- name: Podman Container and Resource Discovery
 <your new atomic>

**Review by patel-bhavin — CHANGES_REQUESTED** [review]

**Comment by patel-bhavin** [comment]
Thank you @W00glin ! We appreciate your first time contribution to this project. There were a few minor updates needed to make this a valid atomic yml. I have fixed up this file and the atomic looks good!

 Incase you are interested in what those changes were : https://github.com/redcanaryco/atomic-red-team/pull/2688/commits/e218a8a77508c029a85712083d069b61bef9cd58

**Review by patel-bhavin — APPROVED** [review]

**Comment by W00glin** [comment]
> Thank you @W00glin ! We appreciate your first time contribution to this project. There were a few minor updates needed to make this a valid atomic yml. I have fixed up this file and the atomic looks good!
> 
> Incase you are interested in what those changes were : [e218a8a](https://github.com/redcanaryco/atomic-red-team/commit/e218a8a77508c029a85712083d069b61bef9cd58)

Glad to help. Thanks for the guidance and feedback. Really! Your contribution guide was one of the easiest to follow along with and made contributing much more simple.

As for [e218a8a](https://github.com/redcanaryco/atomic-red-team/commit/e218a8a77508c029a85712083d069b61bef9cd58), I had considered making that change but that how it was initially laid out, so I left it. Thanks for the clarification.

**Pull request merged**